homeassistant/components/tplink/README.md

```diff
@@ -32,3 +32,8 @@ When doing so, do not forget to run `script/setup` to generate the translations.
 Other information like the category and whether to enable per default are read from the feature,
 as are information about units and display precision hints.
 
+In order to make the tests pass, you will also need to perform the following steps:
```

Comment:
Maybe the rest of the steps could also be done in a playbook style, but I just wanted to add these necessary steps to simplify the process for whomever wants to add the next new entity :-)